build.gradle

```diff
@@ -31,7 +31,7 @@ war {
     inputs.property('version', archiveVersion)
 
     filesMatching("**/bb-manifest.xml") {
-        expand version: archiveVersion
+        expand(version: archiveVersion, java_home: System.properties['java.home'])
```

Comment:
Dat kan niet, want dan krijg je je lokale java home op het moment van builden... Het moet echt als variabele in die bb-manifest belanden, zodat de Blackboard server dat verder in kan vullen.